apiserver/facades/agent/provisioner/provisioninginfo.go

```diff
@@ -112,12 +123,14 @@ func (api *ProvisionerAPI) getProvisioningInfoBase(
 	m *state.Machine,
 	env environs.Environ,
 	endpointBindings map[string]string,
+	modelConfig *config.Config,
+	modelInfo model.ReadOnlyModel,
```

Comment:
We're saving the modelInfo work for the model epic. Leave it for now. Keep in mind for the future.